traefikee/Changelog.md

```diff
@@ -1,10 +1,38 @@
 # Change Log
 
+## 3.1.0  ![AppVersion: v2.10.7](https://img.shields.io/static/v1?label=AppVersion&message=v2.10.7&color=success&logo=) ![Kubernetes: >= 1.14.0-0](https://img.shields.io/static/v1?label=Kubernetes&message=%3E%3D+1.14.0-0&color=informational&logo=kubernetes) ![Helm: v3](https://img.shields.io/static/v1?label=Helm&message=v3&color=informational&logo=helm)
```

Comment:
Why Kubernetes <= v1.14? It doesn't fit the supported version: https://doc.traefik.io/traefik-enterprise/installing/kubernetes/teectl/#requirements

Reply:
Cc @mloiseleur ⬆️  WDYT?

Reply:
Why not. See #122